compiler/src/main/kotlin/io/spine/protodata/Strings.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2021, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+@file:JvmName("Strings")
+
+package io.spine.protodata
+
+/**
+ * Obtains the same string but with the first capital letter.
+ *
+ * If the first char of the string cannot be capitalized (e.g. is not a letter, is already
+ * capitalized, etc.), obtains the same string.
+ */
+public fun String.titleCase(): String =
+    replaceFirstChar { it.titlecase() }
+
+/**
+ * Obtains the same string but in camel case instead of snake case.
+ *
+ * Examples:
+ *  - `"aaa"` becomes `"Aaa"`;
+ *  - `"field_name"` becomes `"FieldName"`;
+ *  - `"TypeName"` stays `"TypeName"`;
+ *  - `"___u_ri____"` becomes `"URi"`.
+ */
+@Suppress("FunctionName") // Demonstrates the CamelCase example.
```

Comment:
Why don't we do it the same way for `TitleCase` then?

Reply:
I think having good KDoc is quite enough. Most people know what a camel case is.